src/components/Nav.js

```diff
@@ -55,63 +64,65 @@ function Nav (props) {
+            bar={selectedItem === '/exec/home' ? barSelected : barUnselected}
           />
           <MenuItem
             label='Create Event'
-            icon={<AddBoxIcon style={selectedItem === '/event/new' ? selected : unselected} />}
-            onClick={handleItemClick.bind(null, '/event/new')}
-            bar={selectedItem === '/event/new' ? barSelected : barUnselected} />
-          {renderDesktopOnly && <MenuItem
-            label='Logout'
-            icon={<Logout />}
-            onClick={logout} />}
-        </React.Fragment>
-        : <React.Fragment>
-          <MenuItem
-            label='Home'
-            icon={<Biztech fill={selectedItem !== '/' && '#fff'} size={ICON_SIZE} />}
-            onClick={handleItemClick.bind(null, '/')}
-            bar={selectedItem === '/' ? barSelected : barUnselected}
+            icon={<AddBoxIcon style={selectedItem === '/exec/event/new' ? selected : unselected} />}
+            onClick={handleItemClick.bind(null, '/exec/event/new')}
+            bar={selectedItem === '/exec/event/new' ? barSelected : barUnselected}
           />
-          <MenuItem
-            label='Events'
-            icon={<DateRangeIcon style={selectedItem === '/events' ? selected : unselected} />}
-            onClick={handleItemClick.bind(null, '/events')}
-            bar={selectedItem === '/events' ? barSelected : barUnselected} />
-          <MenuItem
-            label='Profile'
-            icon={<PersonIcon style={selectedItem === '/profile' ? selected : unselected} />}
-            onClick={handleItemClick.bind(null, '/profile')}
-            bar={selectedItem === '/profile' ? barSelected : barUnselected} />
-          {renderDesktopOnly && <MenuItem
-            label='Logout'
-            icon={<Logout fill={selectedItem !== '/' && '#fff'} size={ICON_SIZE} />}
-            onClick={logout}
-            bar={barUnselected} />}
-        </React.Fragment>
-      }
+          <Divider style={styles.divider} />
+        </>
+        }
+        <MenuItem
+          label='Home'
+          icon={<Biztech fill={selectedItem !== '/' && '#fff'} size={ICON_SIZE} />}
+          onClick={handleItemClick.bind(null, '/')}
+          bar={selectedItem === '/' ? barSelected : barUnselected}
+          featureFlag='REACT_APP_SHOW_MAXVP'
+        />
+        <MenuItem
+          label='Events'
+          icon={<DateRangeIcon style={selectedItem === '/events' ? selected : unselected} />}
+          onClick={handleItemClick.bind(null, '/events')}
+          bar={selectedItem === '/events' ? barSelected : barUnselected}
+          featureFlag='REACT_APP_SHOW_MAXVP'
+        />
+        <MenuItem
+          label='Profile'
+          icon={<PersonIcon style={selectedItem === '/profile' ? selected : unselected} />}
+          onClick={handleItemClick.bind(null, '/profile')}
+          bar={selectedItem === '/profile' ? barSelected : barUnselected}
+          featureFlag='REACT_APP_SHOW_MAXVP'
```

Comment:
is there any reason to use the feature flag on each individual item on the nav bar? would it not be sufficient to do something like: 
`isFeatureEnabled ? // show member nav bar : // do not show member nav bar`

Reply:
There's not a huge reason why... I just felt it was cleaner than having too many ternary operators in the render function...